KNOWN_ISSUES.md

```diff
@@ -0,0 +1,95 @@
+# Swift for TensorFlow Known Issues
+
+This is a curated list of known Swift for TensorFlow issues. With every release,
+new issues are added and resolved issues are updated.
+
+Please see the [JIRA issue tracker](https://bugs.swift.org/projects/TF/issues)
+for a full list of known issues.
+
+## Version 0.3
+
+### TensorFlow Library
+
+* [ ] `Tensor` advanced indexing and striding are not supported on GPU.
+
+## Version 0.2
+
+### Notebook Environments (Colab and Jupyter)
+
+* [ ] When a runtime error occurs or when you interrupt cell execution,
+      resources (e.g. host memory, GPU memory) do not get released. This can
+      lead to OOMs. ([TF-338](https://bugs.swift.org/browse/TF-338))
+  * Workaround: Restart the runtime ("Runtime > Restart Runtime" in the Colab
+    menu bar) to release all the resources.
+* [ ] If the last statement on a cell evaluates to a struct that was defined in
+      the notebook, then you get an error ("use of undeclared type") instead of
+      seeing the value of the statement.
+      ([TF-125](https://bugs.swift.org/browse/TF-125))
+  * Workaround: Wrap the last statement in `print()`.
+* [ ] Using extensions to conform a type to a protocol (e.g. `extension MyType:
+      MyProtocol { ... }`), often causes duplicate conformance errors.
+      ([TF-162](https://bugs.swift.org/browse/TF-162))
+  * Workaround: Go back to the cell where the type is defined, add the
+    conformance to the original declaration, and re-execute the cell.
+* [ ] If a cell that declares a type executes twice, then it creates two
+      different types with the same name. Mixing these types can lead to
+      confusing error messages like "cannot convert value of type 'MyType' to
+      expected argument type 'MyType'".
+      ([TF-156](https://bugs.swift.org/browse/TF-156))
+* [ ] The autocomplete UI needs improvement. It should show types and
+      documentation. It should position your cursor in the first argument when
+      you complete a function call.
+
+### Swift Standard Library Enhancements
+
+* [ ] The
+      [`Differentiable`](https://www.tensorflow.org/swift/api_docs/Protocols/Differentiable)
+      protocol's `allDifferentiableVariables` requirement should not have a
+      setter. Do not use this directly through a generic type with a
+      `Differentiable` conformance constraint.
+      ([TF-208](https://bugs.swift.org/browse/TF-208))
+
+### Swift for TensorFlow Deep Learning Library
+
+* [ ] The `Differentiable` protocol's `allDifferentiableVariables` property
```

Comment:
Dupe (or at least relatively similar) with above. Could we have only one of the two?

Reply:
Thanks! Deleted the item from `Swift for TensorFlow Deep Learning Library`, preserving just the one under `Swift Standard Library Enhancements`.

Reply:
An aside: I considered renaming `Swift Standard Library Enhancements` to `Swift Standard Library` (given that this is the known issues doc) but I feel `Swift Standard Library Enhancements` is more appropriate.